lib/jsonapi/authorization/default_pundit_authorizer.rb

```diff
@@ -190,6 +190,40 @@ def remove_to_many_relationship(source_record, related_record)
       def remove_to_one_relationship(source_record, related_record)
         raise NotImplementedError
       end
+
+      # Any request including <tt>?include=other-resources</tt>
+      #
+      # This will be called for each has_many relationship if the include goes
+      # deeper than one level until some authorization fails or the include
+      # directive has been travelled completely.
+      #
+      # We can't pass all the records of a +has_many+ association here due to
+      # performance reasons, so the class is passed instead.
+      #
+      # ==== Parameters
+      #
+      # * +record_class+ - The underlying record class for the relationships
+      #                    resource.
+      # * +source_record+ — The source relationship record, e.g. an Article in
+      #                     article.comments check
+      def include_has_many_resource(record_class, source_record)
+        ::Pundit.authorize(user, record_class, 'index?')
+      end
+
+      # Any request including <tt>?include=another-resource</tt>
+      #
+      # This will be called for each has_one relationship if the include goes
+      # deeper than one level until some authorization fails or the include
+      # directive has been travelled completely.
+      #
+      # ==== Parameters
+      #
+      # * +related_record+ - The associated record to return
+      # * +source_record+ — The source relationship record, e.g. an Article in
+      #                     article.author check
+      def include_has_one_resource(related_record, source_record)
```

Comment:
I'd swap the order of the arguments here to be more consistent with the rest of the methods.